web-ui/src/helpers/checks.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Full check for whether an array actually exists or is empty, etc
+ * @param arr - an array
+ * @returns a boolean
+ */
+
+export const isArrayPresent = (arr) => Array.isArray(arr) && arr.length;
+
+/**
+ * Check for whether unique object is already in an array and return a boolean.
+ * @param arr - an array
+ * @returns a boolean
+ */
+export const isObjectInArray = (arr, obj) => {
+  return arr.includes(obj);
+};
```

Comment:
This function is not being used anywhere

Reply:
Removed